config/helm/delete-namespaces.yaml

```diff
@@ -0,0 +1,25 @@
+# We deploy everything to giantswarm namespace.
```

Comment:
Patch to delete custom CAPI namespaces, as we deploy everything to `giantswarm` namespace.